app/models/preneeds/form_fill.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+require 'common/models/base'
+require 'preneeds/service'
+
+module Preneeds
+  class FormFill < Common::Base
+    def initialize
+      client = Preneeds::Service.new
+
+      self.attachment_types = client.get_attachment_types.to_h
+      self.branches_of_services = client.get_branches_of_service.to_h
+      self.cemeteries = client.get_cemeteries.to_h
+      self.discharge_types = client.get_discharge_types.to_h
+      self.states = client.get_states.to_h
+    end
+
+    attribute :attachment_types, Array[Preneeds::AttachmentType]
+    attribute :branches_of_services, Array[Preneeds::BranchesOfService]
+    attribute :cemeteries, Array[Preneeds::Cemetery]
+    attribute :states, Array[Preneeds::State]
+    attribute :discharge_types, Array[Preneeds::DischargeType]
+
+    # There is no natural id.
+    def id
+      Digest::SHA2.hexdigest to_json
+    end
+  end
+end
```

Comment:
I would have to insert my own paging, sorting, and filtering for the various collections contained within should the FE decide they needed it? Let me get the Ok from Eugene that he will not need to filter or page cemeteries, and I can change the inheritance to common::Form